### ISSUE TITLE
Practice matches and Official matches

### ISSUE THREAD

**Issue by nallux-dozryl** [ashelkovnykov/urbit-chess#25]
There should be different game modes. Practice matches that do not get recorded and Official matches where the Win/Lose Ratio is recorded.

**Comment by bonbud-macryg** [comment]
This will be easy to add in after #6 and #17 are implemented.

**Comment by bonbud-macryg** [comment]
"Practice" or "Casual" games shouldn't impact ELO ratings either, while "Challenge" or "Official" games should.

**Comment by ashelkovnykov** [comment]
I'm going to remove this from Milestone 1; it's blocked by integration with `%telos`, which is not part of the bounty.

**Comment by ashelkovnykov** [comment]
The way I'm picturing it is that there are 3 types of matches:
1. Rated and recorded
2. Unrated and recorded
3. Unrated and unrecorded

Number 3 is what this issue is requesting: a match that is not only not using the rating server, but also doesn't get added to a user's archive of played games. It's basically the practice board, but with another person.

With that in mind, it's actually not very difficult to add at all: just need a loobean field in `chess-challenge` and `active-game-state` to track that a game shouldn't be added to the archive upon completion.

For that reason, I'm going to re-add this to Milestone 1. However, it's still blocked by #6 and #2.

**Comment by bonbud-macryg** [comment]
Not blocked by #17, but would be easier afterwards.
